lib/websocket_server.rb

```diff
@@ -35,6 +35,9 @@ def call(env)
       return proxy.cleanup if proxy.error
       proxy.start
 
+      # Release the connection because one SPICE console can open multiple TCP connections
+      ActiveRecord::Base.connection_pool.release_connection
```

Comment:
LGTM.  [ConnectionPool#release_connection](https://github.com/rails/rails/blob/107f4282bbfabc011d5ad3bcf3fb3c6fb812ad30/activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb#L276-L278) does exactly what we want:

```
      # Signal that the thread is finished with the current connection.
      # #release_connection releases the connection-thread association
      # and returns the connection to the pool.
```